### DISCUSSION TITLE
Devices In Video Storm Matrix System not connecting to Multicast

### DISCUSSION THREAD

**Discussion by gfaulk09** [bluenviron/mediamtx#2445] [Questions]
### Question

Trying to use MediaMTX with video storm IP HDMI matrix software. We have over 30tvs. Other sources/software multicast throughout our network properly without issue.
It seems that connections to the tvs using MediaMTX are connecting, even through UDP but it is without multicast.

Is there any way to force multicast to every connection? or a different way to enable the multicast in the url?

The URL I am using rtsp://10.1.30.57:8554/mystream?vlcmulticast

**Comment by aler9** [comment]
It's impossible to understand what the problem is without a minimum explanation of what your setup is (what's the source of the streams? what's the final destination? how is MediaMTX involved?) and without server logs or network dumps. Please open another question or a bug and provide requested data.
